### DISCUSSION TITLE
No such file or directory error

### DISCUSSION THREAD

**Discussion by VikramGovindarajan** [paulromano/endf-python#5] [Q&A]
I am a beginner in nuclear data processing. I get the following error when I try to run the following commands after installation.

import endf
mat = endf.Material('n-092_U_235.endf')

FileNotFoundError: [Errno 2] No such file or directory: ''n-092_U_235.endf'

Kindly tell where to find this ENDF file from and where to store it?

Thanks and Regards,
G.Vikram

**Comment by TobiasWegener** [comment]
Hi @VikramGovindarajan, 

The error tells you then the file, you are trying to load, does not exist, or is at least not in the same directory.

So your script (let's name it script.py containing::
```
import endf
mat = endf.Material('n-092_U_235.endf')
```
is in a directory (/nuclear_data) like so:
/nuclear_data/script.py

, then you must download and put your 'endf' file next to it:

/nuclear_data/script.py
/nuclear_data/n-092_U_235.endf

Then try to run the script file again.
Hope that helps.
